samples/snippets/src/main/java/com/example/servicedirectory/EndpointsCreate.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.servicedirectory;
+
+// [START servicedirectory_create_endpoint]
+
+import com.google.cloud.servicedirectory.v1beta1.Endpoint;
+import com.google.cloud.servicedirectory.v1beta1.RegistrationServiceClient;
+import com.google.cloud.servicedirectory.v1beta1.ServiceName;
+import java.io.IOException;
+
+public class EndpointsCreate {
+
+  public static void createEndpoint() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String locationId = "your-region";
+    String namespaceId = "your-namespace";
+    String serviceId = "your-service";
+    String endpointId = "your-endpoint";
```

Comment:
Should this be `displayName` or is it the actual ID used by the API?

Reply:
It is both. ("id" is also what the field is called in the request proto)

Reply:
Huh confusing, but ok. 
Question since I'm not familiar with the API. 

Would it be helpful to add a comment that these are user created or do they already have these configured from something and know what these fields should be?

Reply:
Our resources are hierarchical; you create a namespace in an existing project. You create a service in an already existing namespace. And you create an endpoint in an already existing service. So in this snippet, the project/location/namespace/serviceId should all already exist/ be configured. The endpointId is the only newly created variable to fill here.

This is in our docs pretty explicitly (https://cloud.google.com/service-directory/docs/configuring-service-directory) but I can add a comment here too.

Reply:
👍